src/components/view-pane/index.tsx

```diff
@@ -68,32 +68,29 @@ class ViewPaneBase extends React.PureComponent<ViewPaneProps, {}> {
   }
 
   public render() {
-    const { isQuerySpecific } = this.props;
-    const { manualSpecificationOnly } = this.props.config;
-    const relatedViews = !manualSpecificationOnly && (
-      <div className="pane" styleName={!this.props.relatedViews.isHidden ?
-      "view-pane-related-views" : "view-pane-related-views-hide"}>
+    const {isQuerySpecific, relatedViews} = this.props;
+    const {manualSpecificationOnly, hideRelatedViews} = this.props.config;
+    const hideCondition = (relatedViews.isHidden === undefined && hideRelatedViews) ||
+      (relatedViews.isHidden !== undefined && relatedViews.isHidden);
+    const relatedViewsElement = !manualSpecificationOnly && (
+      <div className="pane" styleName={hideCondition ? "view-pane-related-views-hide" : "view-pane-related-views"}>
         <RelatedViewsButton
-          relatedViews={this.props.relatedViews}
+          hideRelatedViews={hideCondition}
           handleAction={this.props.handleAction}
```

Comment:
This is inconsistent -- why not putting handleAction above in `const {isQuerySpecific, relatedViews} = this.props;`